pyonfx/convert.py

```diff
@@ -61,24 +65,37 @@ def time(ass_ms: Union[int, str]) -> Union[str, int, ValueError]:
             If milliseconds -> ASS timestamp, else if ASS timestamp -> milliseconds, else ValueError will be raised.
         """
         # Milliseconds?
-        if type(ass_ms) is int and ass_ms >= 0:
-            return "{:d}:{:02d}:{:02d}.{:02d}".format(
-                math.floor(ass_ms / 3600000) % 10,
-                math.floor(ass_ms % 3600000 / 60000),
-                math.floor(ass_ms % 60000 / 1000),
-                math.floor(ass_ms % 1000 / 10),
-            )
+        if isinstance(ass_ms, int) and ass_ms >= 0:
```

Comment:
`isinstance` is not ideal in these cases.
Considering this exact case, we do not want to check for `ass_ms` to be `int` or any subtype of `int`, we do want to check for `ass_ms` to be exactly `int`.

Reply:
I completely understand what you are saying but why the user would use a subclass of `int` in the first place?

Reply:
Exactly. There is no need for the user to use a subclass of int, so it is pointless to use "isinstance". We would like to check specifically for "int".